admin/class-froala-admin.php

```diff
@@ -345,7 +347,7 @@ public function froala_editor ($settings, $editor_id = null) {
 		   \'imageUploadParams\': {\'action\' : \'froala_upload_files\'},
 		   \'imageManagerLoadParams\':{\'action\' : \'froala_image_manager\'}});
 		   }); </script>' . "\n";
-
+											  
```

Comment:
@Geet06 What is this change for?

Reply:
@kapil2704 an extra empty got added. Removed it